src/pybind/ceph_volume_client.py

```diff
@@ -660,8 +664,14 @@ def deauthorize(self, volume_path, auth_id):
         pool_name = self._get_ancestor_xattr(path, "ceph.dir.layout.pool")
         namespace = self.fs.getxattr(path, "ceph.dir.layout.pool_namespace")
 
-        want_mds_cap = 'allow rw path={0}'.format(path)
-        want_osd_cap = 'allow rw pool={0} namespace={1}'.format(pool_name, namespace)
+        # The auth_id might have read-only or read-write mount access for the
```

Comment:
As @xiaoxichen suggested , I think it'd be better to deauthorize based on access_level.
So the deauthorize function's signature would be,
deauthorize(self, volume_path, auth_id, access_level='rw').
'access_level=rw' to be compatible with the Mitaka version of cephfs_native driver.